### ISSUE TITLE
Add dropdown-list component like select tag

### ISSUE THREAD

**Issue by neosarchizo** [nodegui/react-nodegui#70]
**Is your feature request related to a problem? Please describe.**
Please add a dropdown-list component like select tag.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by master-atul** [comment]
I think https://doc.qt.io/qt-5/qcombobox.html is what you are looking for. 
If you can help out in porting it in nodegui first that would be awesome!
I can help you out. Otherwise, let me know, I ll see if I can squeeze it in the next release. @neosarchizo

**Comment by neosarchizo** [comment]
I folked [nodegui](https://github.com/nodegui/nodegui), and checked codes of components. But wow.. there are so many things to do for adding new component. 

- Make wrapper c codes for QComboBox
- Make JS library for QComboBox
- Make React library for QComboBox

I really wanna do that but I don't have enough time to do that. I'm really sorry.
But I really like your project, so I can wait for next release.
Thanks!

**Comment by master-atul** [comment]
closed by #93

**Comment by gregbenner** [comment]
Is there an example... 
```<ComboBox>
          <ComboBoxItem text="hi"></ComboBoxItem>
          <ComboBoxItem text="by"></ComboBoxItem>
        </ComboBox>``` ?
